sdks/python/apache_beam/io/source_test_utils_test.py

```diff
@@ -115,6 +131,9 @@ def test_split_at_fraction_binary(self):
     self.assertTrue(stats.successful_fractions)
     self.assertTrue(stats.non_trivial_fractions)
 
+  @unittest.skipIf(sys.version_info[0] == 3, 'This test still needs to be '
```

Comment:
This PR has a lot of tests excluded due to `BEAM-5627`,  I assume that you have verified that these excluded tests have the symptom described by `BEAM-5627`. If they fail for a different reason, we should use a different Jira.

Reply:
Using the new env var I was able to verify this, and this is in fact a highly recurring error. Only this particular error has the Jira code.